CONTRIBUTING (#1074)
vtex/styleguide: .github/CONTRIBUTING.md

# Contributing to VTEX Styleguide

Want to contribute to VTEX Styleguide? There are a few things you need to know.

### Release Schedule

> Thursday release: patch version at the thursday of every week for routine bugfix (anytime for urgent bugfix).

Reviewer: ```suggestion
Thursday release: patch version at the Thursday of every week for routine bug fix (anytime for urgent bug fix).
```

> Thursday release: patch version at the Thursday of every week for routine bug fix (anytime for urgent bug fix).

Reviewer 2: ```suggestion
Thursday release: patch version weekly on each Thursday for routine bug fix (anytime for urgent bug fix).
```

Author: make sense

> Tuesday release: minor version at the tuesday of every month for new features.

Reviewer 3: ```suggestion
Tuesday release: minor version at the tuesday of every week for new features.
```

Reviewer 3: Or is it a specific tuesday?

Author: is weekly

> Tuesday release: minor version at the tuesday of every month for new features.

Reviewer: ```suggestion
Tuesday release: minor version at the Tuesday of every month for new features.
```

Reviewer: Month or week?

> Major version release is not included in this schedule for breaking change and new features.

Reviewer 3: ```suggestion
Major version release with possible breaking-changes is not included in this schedule.
```

Reviewer 3: I'm not not the best fan of my change also, but the original is kind of weird too 🤔

Author: @klzns' suggestion was more pleasant.

Author: anyway thank you!

> Major version release is not included in this schedule for breaking change and new features.

Reviewer: What do you think of just:

```suggestion
Major version releases are not included in this schedule.
```

### Open Development
All work on VTEX Styleguide happens directly on GitHub. Both core team members and external contributors send pull requests which go through the same review process.

### Branch Organization

> According to our [release schedule](#release-schedule), we maintain two branches, `master` and `dev`. If you send a bugfix pull request, please do it against the `master` branch, if it's a feature pull request, please do it against the `dev` branch.

Reviewer 4: ```suggestion
According to our [release schedule](#release-schedule), we maintain two branches: `master` and `dev`. Bug fixes should have `master` as it base branch, while new features should be merged into `dev`.
```

Reviewer 4: Just to be more direct and avoid repetitions

Author: done!
